cloudflare/resource_cloudflare_filter.go

```diff
@@ -40,6 +40,10 @@ func resourceCloudflareFilter() *schema.Resource {
 			"expression": {
 				Type:     schema.TypeString,
 				Required: true,
+				// Suppress the diff shown if there's trailing carriage returns at the end of the filter (needed for heredocs)
```

Comment:
I would rather we remove the inline comment and just have this information (such as the driver) in the commit message itself.

```suggestion
```